utils/gpsd/patches/010-musl.patch

```diff
@@ -1,40 +0,0 @@
---- a/driver_ais.c
```

Comment:
I don't think this patch should be removed. It fixes compilation when compiling with -Werror=implicit-function-declaration.

Reply:
Never mind. Just saw that's it not needed anymore.